server/rest/src/main/java/org/infinispan/rest/logging/Messages.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.rest.logging;
+
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+/**
+ * Informational Scripting messages. These start from 12500 so as not to overlap with the logging
```

Comment:
```suggestion
 * Informational Rest Server messages. These start from 12500 so as not to overlap with the logging
```

is there a document with the Message's ids? Some `Messages` classes have ids, other don't, others have messages with and without id (`org.infinispan.util.logging.events.Messages`).

Reply:
They use the same ranges as the logging ids and those are defined in the contribution guide